zerver/openapi/zulip.yaml

```diff
@@ -14009,10 +14037,21 @@ paths:
                                 on the external site.
                       jitsi_server_url:
                         type: string
+                        deprecated: true
                         description: |
                           Present if `realm` is present in `fetch_event_types`.
 
-                          The base URL the organization uses to create Jitsi video calls.
+                          The base URL to be used to create Jitsi video calls. Equals
+                          `realm_jitsi_server_url || server_jitsi_server_url`.
+
+                          **Changes**: Deprecated in Zulip 8.0 (feature level 212) and will
+                          eventually be removed. Previously, the Jitsi server to use was not
+                          configurable on a per-realm basis, and this field contained the server's
+                          configured Jitsi server. (Which is now provided as
+                          `server_jitsi_server_url`). Clients supporting older versions should fall
+                          back to this field when creating calls: using `realm_jitsi_server_url ||
+                          server_jitsi_server_url` with newer servers and using `jitsi_server_url`
+                          with servers below feature level 212.
```

Comment:
This can also be written as a double fallback, like `realm_jitsi_server_url ?? server_jitsi_server_url ?? jitsi_server_url`.

In particular that's how I think I'd want to write it in client code. Typically I've preferred a fallback over a version check, where possible; the code feels more robust that way, less sensitive to the exact details in the API docs, and also more transparent to the reader without having to go look up the API docs.

Reply:
No, I meant what I wrote: The double fallback is I think wrong -- in the unlikely corner case of a modern server that starts out with `server_jitsi_server_url=None` and `realm_jitsi_server_url` set to `jitsi.example.com`, and one changes `realm_jitsi_server_url` to None, then `jitsi_server_url` will still have `jitsi.example.com`.

Reply:
I see. I think I wasn't considering the possibility of the server-level value being null while the realm-level value isn't.